### PR TITLE
Add images and galleries for new pages

### PR DESCRIPTION
**Pull request by joaojusto**
Why:

* We need to add routes and copy for the [new
pages](https://paper.dropbox.com/doc/From-Kibera-with-Love-website-oGDL0WZUa2ZWHdcL1MMC3#:h2=Patrocinadores-e-Parceiros);

These changes address the need by:

* Adding new assets under `/src/assets`;
* Refactoring `<Gallery/>` so that it exports different sections;
* Exporting a gallery for each section;
* Adding each new `<Gallery/>` to it's correspondent page;
* Other small changes;

### CODE REVIEW

**Review by jfranciscosousa — APPROVED** [review]
Looks good

**Comment by joaojusto** [comment]
@jfranciscosousa is there any other way to do these queries  besides doing one for each path? :thinking: :disappointed:

**Comment by jfranciscosousa** [comment]
@joaojusto you can take all paths and filter by filename

File: src/components/Gallery/activities.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Gallery from './gallery.js'
+import { graphql, useStaticQuery } from 'gatsby';
+
+export default () => {
+  const {
+    allFile: { edges: images }
+  } = useStaticQuery(graphql`
+    query {
+      allFile(filter: { absolutePath: { regex: "assets/about/activities/" } }) {
```
Comment (jfranciscosousa) [inline src/components/Gallery/activities.js:10]: Change the `regex` to `assets/about/`and filter with Javascript maybe

**Pull request closed without merging**